homeassistant/components/tibber/sensor.py

```diff
@@ -373,20 +368,21 @@ async def _fetch_data(self):
         ]["estimatedAnnualConsumption"]
 
 
-class TibberSensorRT(TibberSensor):
+class TibberSensorRT(update_coordinator.CoordinatorEntity, TibberSensor):
```

Comment:
Normally it's better to put the integration specific entity class first in the MRO.